src/components/CurriculumComponents/OverviewTab/OverviewTab.tsx

```diff
@@ -307,14 +326,6 @@ const OverviewTab: FC<OverviewTabProps> = (props: OverviewTabProps) => {
                 $maxWidth={["100%", "100%", "65%"]}
                 $textAlign={"left"}
               >
-                <OakHeading
```

Comment:
You've removed this heading but it should instead be visually hidden like the Cycle 2 enabled version above.

Reply:
Reinstated